lib/util/memcache.ex

```diff
@@ -6,10 +6,50 @@ defmodule Antikythera.Memcache do
   @moduledoc """
   In-memory cache for each executor pool.
 
-      iex> Antikythera.Memcache.write("foo", "bar", epool_id)
+  Antikythera provides in-memory cache controlled by AntikytheraCore for each executor pool.
+  Objects are stored in ETS internally.
+
+  ## Usage
+
+  Memcache behaves as a key-value storage.
```

Comment:
To me, treating "memcache" as a general noun feels not just right, might be better to use module name explicitly?
In that, putting `#{inspect(__MODULE__)}` is handy since it's rendered as "Antikythera.Memcache" in ex_doc.

```
#{inspect(__MODULE__)} behaves as a key-value storage.
```

Also applies to later parts.

Reply:
Fixed in e8adcb50f657f23724421e7b0383d11d95fe9bb0